src/components/measuresDetails.jsx

```diff
@@ -0,0 +1,109 @@
+import { graphql } from "gatsby";
+import React from "react";
+
+import { TotalAmountPanelItem } from "../components/totalAmountItem";
+import LandingPageHero from "../components/landingPageHero";
+import Layout from "../components/layout";
+import SideNav from "../components/sideNav";
+import SectionHeader from "../components/sectionHeader";
+import styles from "../templates/candidate.module.scss";
+import useWindowIsLarge from "../common/hooks/useWindowIsLarge";
+import VotersEdgeIcon from "../../static/images/votersEdge.png";
+import ChartSection from "./ChartSection";
+
+const supportingCommittees = [
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 500000,
+  },
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 400000,
+  },
+  {
+    title: "Yes on Measure E! San José Neighbors for Parks & People",
+    value: 14000,
+  },
+];
+
+
+function MeasureDetails(props) {
+  const measure =
+    props.data?.allElection?.edges?.[0]?.node?.Referendums?.[0] ?? {};
```

Comment:
So, I tried to use the nullish coalescing operator before and it broke on other people's machines (not sure why it didn't break on mine). This might have been fixed by updating gatsby, but if not then we might need to add a custom .babelrc if we want to use it: https://www.gatsbyjs.com/docs/babel/#how-to-use-a-custom-babelrc-file

I haven't tried optional chaining, but we might have a similar issue there? Anyway, I'd say go for it and then we can fix it forward with our own .babelrc if we get issues :)